documentation/sphinx/source/release-notes/release-notes-7131.rst

```diff
@@ -0,0 +1,198 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.1
+======
+
+Features
```

Comment:
Most of these are improvements to features added in 71.3.0 rather than new features. Would it make sense to separate them out into a "Feature Improvements" section?

Reply:
I scanned through the PRs and constructed this list, if we know the specific one that needs to be filtered, happy to separate them out.

Reply:
I'm happy to suggest the ones that I think are improvements, but in keeping with the consistency theme, since there isn't a "Feature Improvements" section, I think they probably fit better in the "Features" section rather than "Other Changes" section, so let's leave it as is.